L1Trigger/TrackFindingTracklet/interface/ProjectionCalculator.h

```diff
@@ -5,6 +5,8 @@
 #include "L1Trigger/TrackFindingTracklet/interface/TrackletLUT.h"
 #include "L1Trigger/TrackFindingTracklet/interface/TrackletProjectionsMemory.h"
 #include "L1Trigger/TrackFindingTracklet/interface/TrackletParametersMemory.h"
+#include "L1Trigger/TrackFindingTracklet/interface/TrackletCalculatorBase.h"
+#include "L1Trigger/TrackFindingTracklet/interface/TrackletConfigBuilder.h"
```

Comment:
Does these need to be in the header file?

Reply:
TrackletCalculatorBase.h is not necessary and will be removed, but TrackletConfigBuilder.h is to use the iMergedTCStr function to get the page the projections need to be written to (this seemed the best way to do this).